openmrs/startup.sh

```diff
@@ -35,6 +30,22 @@ echo "-----------------------------------"
 # wait for mysql to initialise
 /usr/local/tomcat/wait-for-it.sh --timeout=3600 ${DB_HOST}:3306
 
+
+mysql_result=`mysql --defaults-extra-file=/tmp/db_credentials.txt -h${DB_HOST} --skip-column-names -e "SELECT count(*) FROM information_schema.tables WHERE table_schema = '${DB_DATABASE}'"`
+
+if [ ${mysql_result} > 1 ]; then
+    cat > /usr/local/tomcat/.OpenMRS/openmrs-runtime.properties << EOF
+encryption.vector=KOWPyyXpnQfFWo2p2Cl8Zg\=\=
```

Comment:
Is this going to be a security threat? as well as `encryption.key` below.
Should we generate them on the fly?

Reply:
@chinzou , see this: https://talk.openmrs.org/t/encryption-vector-and-key-in-openmrs-runtime-properties/26472?u=mksrom

Reply:
I will check what's the result if we don't set the `encryption.key`